keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -0,0 +1,1116 @@
+          namespaceSelector: 
+            matchLabels: {}
+```
+
+#### Sample spec for Story 2: Allow traffic at a cluster level
+
+```yaml
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata: 
+  name: cluster-wide-allow-example
+spec:
+  priority: 30
+  - appliedTo:
+     namespaceSelector: {}
+    ingress:
+     - action: Allow
+       from:
+       - namespaceSetSelector: 
+          scope: Selector
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: monitoring-ns
+    egress: 
+      - action: Allow
+        to: 
+        - namespaceSetAndPodSelector:
+            scope: Selector
+            namespaceSelector:
+              matchlabels: 
+                kubernetes.io/metadata.name: kube-system 
+            podSelector: 
+              matchlabels: 
+                app: kube-dns
+```
+
+#### Sample spec for Story 3: Explicitly Delegate traffic to existing K8s Network Policy
+
+```yaml
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata: 
+  name: pub-svc-delegate-example
+spec:
+  priority: 20
+  - appliedTo:
+      namespaceSelector: {}
+    egress:
+    - action: Pass
+      to:
+      - namespaceSetAndPodSelector:
+          scope: Selector 
+          namespaceSelector: 
+            matchLabels:
+              kubernetes.io/metadata.name: bar-ns-1 
+          podSelector: 
+            matchLabels:
+              app: svc-pub
+```
+
+#### Sample spec for Story 4: Create and Isolate multiple tenants in a cluster
+
+```yaml
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata: 
+  name: tenant-creation-example
+spec:
+  priority: 50
+  - appliedTo:
+     namespaceSelector:
+       matchExpressions: {key: "tenant"; operator: Exists}
+    ingress:
+     - action: Deny
+       from:
+       - namespaceSetSelector:
+          scope: NotSameLabels
+          labels:
+            - tenant
+```
+
+Note: the above AdminNetworkPolicy can also be written in the following fashion:
+```yaml
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata: 
+  name: tenant-creation-example
+spec:
+  priority: 50
+  - appliedTo:
+     namespaceSelector:
+       matchExpressions: {key: "tenant"; operator: Exists}
+    ingress:
+     - action: Pass
+       from:
+       - namespaceSetSelector:
+          scope: SameLabels
+          labels:
+            - tenant
+     - action: Deny   # Deny everything else other than same tenant traffic
+       from:
+       - namespaceSetSelector:
+          scope: Selector
+          namespaceSelector: {}
+```
+The difference is that in the first case, traffic within tenant Namespaces will fall
+through, and be evaluated against lower-priority ClusterNetworkPolicies, and then
+NetworkPolicies. In the second case, the matching packet will skip all AdminNetworkPolicy
+evaluation (except for AdminNetworkPolicy priority=baseline), and only match
```

Comment:
I think `priority=baseline` has been removed in this iteration, yeah?

Reply:
Yep good catch, done.